bootstrap/helpers/docker.php

```diff
@@ -188,22 +188,39 @@ function get_port_from_dockerfile($dockerfile): ?int
     return null;
 }
 
-function defaultLabels($id, $name, $pull_request_id = 0, string $type = 'application', $subType = null, $subId = null)
+function defaultDatabaseLabels($database) {
+    $labels = collect([]);
+    $labels->push('coolify.managed=true');
+    $labels->push('coolify.type=database');
+    $labels->push('coolify.databaseId='.$database->id);
+    $labels->push('coolify.resourceName='.Str::slug($database->name));
+    $labels->push('coolify.serviceName='.Str::slug($database->name));
```

Comment:
Why add a service name for a database?

Reply:
To be consistent with the others containers that have a `coolify.serviceName` label. 

This way, we can rely solely on `coolify.serviceName` everywhere (for example, in our PromQL queries) to fetch metrics about a container. If we don’t add `coolify.serviceName`, we’ll be forced to write tedious conditions everywhere: checking for the presence of `coolify.serviceName`, and if it’s not available, falling back to `coolify.resourceName`.

Also that's why I said `coolify.serviceName` might not be the best name. Maybe we can find something better